src/pages/Mobile/About/About.tsx

```diff
@@ -1,10 +1,13 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useEffect } from 'react';
 import BackHeader from '../../../components/BackHeader/BackHeader';
 import DescriptionBody from '../../../components/DescriptionBody/DescriptionBody';
 import Header from '../../../components/Header';
 import './About.scss';
 
 function About(): ReactElement {
+    useEffect(() => {
+        document.title = 'Unity Sports: About Us';
+    }, []);
```

Comment:
don't repeat this in every file, make it top level in App.ts. Use the url name in your title so you don't have to rewrite the code. Access the url name.